docs/api-markdown-documenter/render-api-documentation.js

```diff
@@ -55,19 +55,12 @@ async function renderApiDocumentation(inputDir, outputDir, uriRootDir, apiVersio
 		includeBreadcrumb: false, // Hugo will now be used to generate the breadcrumb
 		includeTopLevelDocumentHeading: false, // This will be added automatically by Hugo
 		createDefaultLayout: layoutContent,
-		packageFilterPolicy: (apiPackage) => {
-			// Skip `@fluid-internal` packages
+		skipPackage: (apiPackage) => {
+			// Skip `@fluid-internal` and `@fluid-private` packages
 			const packageName = apiPackage.displayName;
 			const packageScope = PackageName.getScope(packageName);
 
-			console.log(`${packageName}: ${packageScope}`);
-
-			return ["@fluid-internal"].includes(packageScope);
-		},
-		fileNamePolicy: (apiItem) => {
```

Comment:
This is equivalent to the default policy anyways, so this rule override could be removed.